packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/main/index.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Share a sign-in session with native mobile apps
+meta:
+  - name: description
+    content: Mobile apps use an external user-agent (the device's browser) to perform authentication. Get information on how you can apply sign-in principles that are similar to web apps to share a sign-in session between mobile apps on a device.
+layout: Guides
+---
+
+This guide uses sample apps to demonstrate how to share a Single Sign-On (SSO) session on a mobile device. After you complete this guide, you should have a better understanding of how a sign-in session is shared between two mobile apps on a device and will be able to use the steps in the guide to help you configure your own apps.
+
+---
+
+**Learning outcomes**
+
+* Persist a session between multiple OIDC mobile apps
+* Clear the session when appropriate
+
+**What you need**
+
+* An Okta Developer Edition organization. Don't have one? [Create one for free](https://developer.okta.com/signup)
+* Android Studio with an emulator for Android testing
+* Xcode with a simulator for iOS testing
+
+**Sample code**
+
+See the [Sample code](#sample-code) section.
```

Comment:
Re-do **Sample code** section based on our discussion with stack-selectors and nutrition fact template?